<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthFrame</class>
 <widget class="QFrame" name="AuthFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Authentification</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Card number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_cardNum">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="maxLength">
    <number>19</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_password">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AuthFrame</receiver>
   <slot>requestForAuth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>requestForAuth()</slot>
 </slots>
</ui>
